data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,406 @@
+################################################################################################################################################################################
+# AUDIO
+################################################################################################################################################################################
+//--------------------
+// MUSIC
```

Comment:
Using MarkDown would be better than a .txt file with arbitrary heading formatting